### ISSUE TITLE
Can nebula do site to site vpn ?

### ISSUE THREAD

**Issue by ccurlv** [slackhq/nebula#238]
like this [link ](https://campus.barracuda.com/resources/attachments/image/14320455/2/s_to_s_dynamic.png)  
i have 2 DC need transfer data ,  but the test is useless  
I have tried:  
Credential use -subnet, config use unsafe_routes  
sysctl.conf add ip_forward on both sides  
But still can't ping the VM behind nebula

**Comment by hasturo** [comment]
As far as i know you need at least one Lighthouse that is reachable from both Networks. Hosts on both Sides discover their direct encryption Partner through Lighthouse and setup a direct link for encryption. There is no "behind" Nebula like in Classic S2S VPN.

**Comment by ibehren1** [comment]
Yes, it can with unsafe_routes.  You use the unsafe_routes to advertise CIDR blocks that can be accessed and routed through that node.

See here in config file:  https://github.com/slackhq/nebula/blob/master/examples/config.yml#L143

Also use the -subnets option when signing the cert of the node(s) that will host unsafe_routes.
